src/Event.cc

```diff
@@ -189,22 +189,12 @@ void EventMgr::Describe(ODesc* d) const
 
 void EventMgr::Process()
 	{
-	// If we don't have a source, or the source is closed, or we're
```

Comment:
The reason why this is an IO source is so that we can add the file descriptors from the flares to the kqueue.